stdlib/concurrent/futures/process.pyi

```diff
@@ -31,13 +31,11 @@ _MAX_WINDOWS_WORKERS: int
 class _RemoteTraceback(Exception):
     tb: str
     def __init__(self, tb: TracebackType) -> None: ...
-    def __str__(self) -> str: ...
 
 class _ExceptionWithTraceback:
     exc: BaseException
     tb: TracebackType
     def __init__(self, exc: BaseException, tb: TracebackType) -> None: ...
-    def __reduce__(self) -> str | tuple[Any, ...]: ...
```

Comment:
Not 100% sure about this since I could imagine a type checker checking whether an object can be pickled safely and checking `__reduce__` overrides.

Reply:
You've [changed your tune](https://github.com/python/typeshed/pull/6321#issuecomment-972559436) ;)

Reply:
I guess I just default to not making changes when we don't have a clear reason for change :)

Reply:
A sound policy.